agate-webapp/src/main/resources/i18n/en.json

```diff
@@ -535,6 +535,8 @@
     "new-realm": "New Realm",
     "users-help": "List of users authenticated by this realm.",
     "users-none": "No users are authenticated by this realm.",
+    "username-claim": "Username Claim",
+    "username-claim-help": "Key used to get the username from the JWT custom claims.",
```

Comment:
Not exactly... 
1. the claim is also looked up in the UserInfo object
2. it is optional
3. if not found, lookup continues with the claims `preferred_username` etc.